integration-test/src/main/java/uk/gov/gchq/gaffer/integration/impl/ImportExportIT.java

```diff
@@ -96,23 +95,22 @@ public void shouldImportFromFileThenExportBackOutToDifferentFile() throws Operat
                         .build())
                 .build();
 
+        // When
         final Iterable<? extends String> exportedData = graph.execute(importExportOpChain, getUser());
-
-        //   When
-
         Iterable<String> incomingFile = readFile(INCOMING_FILE_PATH);
         Iterable<String> outgoingFile = readFile(path.toFile().getAbsolutePath());
 
 
-        //  Then
+        // Then
+        assertThat(expectedData).containsExactlyInAnyOrderElementsOf(exportedData);
```

Comment:
Might be better if it was asserting that the exported data contained the expected data and not the other way around, as normally we assert that what we have contains what we expect.

Reply:
Managed to fix and get this in